hrm/department_app/models/employees.py

```diff
@@ -0,0 +1,18 @@
+"""Django models"""
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+
+class Employees(models.Model):
+    """Employees"""
+    Id = models.IntegerField('ID')
```

Comment:
Please use AutoField instead of IntegerField for primary keys.

Reply:
Fixed.

Reply:
Why did you remove the field id? I just asked to use correct field type